src/renderer/components/ft-channel-bubble/ft-channel-bubble.vue

```diff
@@ -6,10 +6,16 @@
     :to="`/channel/${channelId}`"
   >
     <img
+      v-if="channelThumbnail !== CHANNEL_IMAGE_BROKEN"
```

Comment:
Yes, I used a constant because there are two different types of non-existent thumbnails:

- "CHANNEL_IMAGE_BROKEN" - invalid channels/banned channels
- "CHANNEL_IMAGE_NOT_EXISTENT" - we haven't fetched the channel's thumbnail yet (we only fetch these channels, CHANNEL_IMAGE_BROKEN can be refreshed by going to the channel's page if the channel isn't broken)